FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -53,6 +53,12 @@
 NSTimeInterval timeoutSeconds = 4320;
 NSInteger FETCH_ATTEMPTS = 3;
```

Comment:
Please check the Objective C style guide. I think all caps is only for #define constants.

Reply:
Updated variables to follow obj-c style guide.